site/js/app.js

```diff
@@ -27,6 +27,7 @@ document.querySelector(".js-flooper-resume").onclick = () => {
 };
 
 // Add random delay to floating animation
-document.querySelectorAll(".js-flooper-item").forEach( (e) => {
-  e.style.animationDelay = Math.random()*3 + "s";
-});
+var elements = document.querySelectorAll(".js-flooper-item");
```

Comment:
FYI: forEach is supported on IE... on arrays. `.querySelectorAll` returns a nodeList, that's a little bit different. That's why is best to convert nodeList to array with `Array.from(nodeList)` or `[].slice.call(nodeList)` or if you're feeling cool, using spread operator `[...nodeList]`.

That being said, it's fine as it is for demo.